octobot/factories/exchange_factory.py

```diff
@@ -31,40 +32,24 @@ def __init__(self, octobot, ignore_config=False):
         self.octobot = octobot
         self.ignore_config = ignore_config
 
-        # Logger
         self.logger = get_logger(self.__class__.__name__)
 
-        self.exchange_manager = None
         self.exchange_manager_ids = []
-        self.exchange_traders = {}
-        self.exchange_trader_simulators = {}
-        self.exchange_trading_modes = {}
-        self.trading_mode = None
-        self.previous_trading_state_manager = None
-        self.exchanges_list = {}
-        self.global_updaters_by_exchange = {}
-
-        self.available_exchanges = ccxt.exchanges
 
     async def _create_exchanges(self, backtesting=None):
         for exchange_class_string in self.octobot.config[CONFIG_EXCHANGES]:
-            if exchange_class_string in self.available_exchanges:
-                exchange_builder = create_exchange_builder(self.octobot.config, exchange_class_string) \
-                    .has_matrix(self.octobot.evaluator_factory.matrix_id) \
-                    .use_tentacles_setup_config(self.octobot.tentacles_setup_config) \
-                    .set_bot_id(self.octobot.bot_id) \
-                    .is_rest_only()
-                if is_trader_enabled_in_config(self.octobot.config):
-                    exchange_builder.is_real()
-                elif is_trader_simulator_enabled_in_config(self.octobot.config):
-                    exchange_builder.is_simulated()
-                if backtesting is not None:
-                    exchange_builder.is_backtesting(backtesting)
-                exchange_manager = await exchange_builder.build()
-                await init_exchange_chan_logger(exchange_manager.id)
-                self.exchange_manager_ids.append(exchange_manager.id)
-            else:
-                self.logger.error(f"{exchange_class_string} exchange not found")
+            await get_chan_at_id(OctoBotChannelsName.OCTOBOT_CHANNEL.value, self.octobot.bot_id)\
```

Comment:
should we add a requirement PR to handle this ?

Reply:
Yes :+1: